<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayersManager</class>
 <widget class="QMainWindow" name="PlayersManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="botLabel">
       <property name="text">
        <string>Bot</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playerLabel">
       <property name="text">
        <string>Player</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Player_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>88</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 6</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
